setup.sh

```diff
@@ -1,34 +1,43 @@
 #!/bin/sh
 
+# script can executed remotely with
+# "bash <(curl -s https://raw.githubusercontent.com/madslundt/cloud-media-scripts/master/setup.sh) 
+
 ########## CONFIGURATION ##########
+wget https://raw.githubusercontent.com/madslundt/cloud-media-scripts/master/config #pulls latest config from master
 . "./config"
 ###################################
 ########## DOWNLOADS ##########
 # Rclone
-_rclone_url="https://github.com/ncw/rclone/releases/download/v1.36/rclone-v1.36-linux-amd64.zip"
-_rclone_zip="rclone-v1.36-linux-amd64.zip"
-_rclone_dir="rclone-v1.36-linux-amd64"
+_rclone_url="https://downloads.rclone.org/rclone-current-linux-amd64.zip" #current stable rclone release
```

Comment:
I like having the version in there so I know this is version XX. New version could contain new bugs or commands could be removed so the scripts don't work anymore.